content/webapp/services/prismic/fetch/pages.ts

```diff
@@ -40,10 +43,45 @@ const fetchLinks = pagesFields.concat(
 );
 
 /** Although these are three different document types in Prismic, they all get
-  * rendered (and fetched) by the same component.
-  */
-const pagesFetcher = fetcher<PagePrismicDocument>(['pages', 'guides', 'projects'], fetchLinks);
+ * rendered (and fetched) by the same component.
+ */
+const pagesFetcher = fetcher<PagePrismicDocument>(
+  ['pages', 'guides', 'projects'],
+  fetchLinks
+);
 
 export const fetchPage = pagesFetcher.getById;
 export const fetchPages = pagesFetcher.getByType;
 export const fetchPagesClientSide = pagesFetcher.getByTypeClientSide;
+
+export const fetchChildren = async (
+  client: GetServerSidePropsPrismicClient,
+  page: Page
+): Promise<PagePrismicDocument[]> => {
+  const predicates = [prismic.predicate.at('my.pages.parents.parent', page.id)];
+
+  try {
+    return await fetchPages(client, { predicates }).then(q => q.results);
+  } catch (e) {
+    console.warn(`Error trying to fetch children on page ${page.id}: ${e}`);
+    return [];
+  }
+};
+
+export const fetchSiblings = async (
+  client: GetServerSidePropsPrismicClient,
+  page: Page
+): Promise<SiblingsGroup<PagePrismicDocument>[]> => {
+  const relatedPagePromises =
+    page.parentPages &&
+    page.parentPages.map(parentPage => fetchChildren(client, parentPage));
+  const relatedPages = await Promise.all(relatedPagePromises);
```

Comment:
Won't this error if there weren't any `parentPages` (as above)?
```suggestion
  const relatedPages = await Promise.all(relatedPagePromises ?? []);
```